docs/fitting_types.rst

```diff
@@ -385,6 +385,33 @@ are added squared, ready to be minimized. Unlike in the above example, the
 
   Do not cite the overall :math:`R^2` given by :mod:`symfit`.
 
+Fitting multidimensional datasets
+--------------------------------
+Imagine that you have now a specific property over a grid, like the surface height,
+so you have a three-dimensional data set. To fit a function with two parameters
+to this data set, you can use :mod:`symfit`.
+
+Let's work this out with the following example. We made a polynomial function
+with two coefficients, representing two terms of mixed order in ``x`` and ``y``
```

Comment:
Forgot a `:` at the end of the sentence.

Reply:
Done :D